### PR TITLE
Added two functions to deal with split daemon mode

### PR DESCRIPTION
**Pull request by iccaszhulili**
As under the split daemon mode, the server name and
config file changed

Signed-off-by: Lily Zhu <lizhu@redhat.com>

### CODE REVIEW

**Comment by dzhengfy** [comment]
@iccaszhulili Please rebase your branch and push it again forcely. The centos 7 failures has been fixed already.

File: virttest/virt_admin.py [hunk 1]
```diff
@@ -702,6 +704,25 @@ def command(cmd, **dargs):
...
+
+
+def managed_daemon_config():
+    """
+    Determine different daemon config under different daemon mode.
+    """
+    if utils_split_daemons.is_modular_daemon():
+        config = utils_config.VirtProxydConfig()
+    else:
+        config = utils_config.LibvirtdConfig()
```
Comment (Yingshun) [inline virttest/virt_admin.py:722]: Same as above, how about setting a default value to conf_type or something else and then get the config object?
def managed_daemon_config(conf_type="virtproxyd"):
    if not utils_split_daemons.is_modular_daemon():
        conf_type = “libvirtd"
    config = utils_config.get_conf_obj(conf_type)
    return config

Comment (iccaszhulili) [reply virttest/virt_admin.py]: @Yingshun updated. Thanks

File: virttest/virt_admin.py [hunk 2]
```diff
@@ -702,6 +704,25 @@ def command(cmd, **dargs):
     return ret
 
 
+def check_server_name():
+    if utils_split_daemons.is_modular_daemon():
+        server_name = "virtproxyd"
```
[the remark this answers is not in the document]
Comment (iccaszhulili) [reply virttest/virt_admin.py]: @Yingshun updated

File: virttest/virt_admin.py [hunk 3]
```diff
@@ -702,6 +704,25 @@ def command(cmd, **dargs):
...
+    """
+    if not utils_split_daemons.is_modular_daemon():
+        server_name = "libvirtd"
+    return server_name
+
+
+def managed_daemon_config(conf_type="virtproxyd"):
+    """
+    Determine different daemon config under different daemon mode.
+    """
```
Comment (Yingshun) [inline virttest/virt_admin.py:719]: Would you please update docstring for parameter and return value?

File: virttest/virt_admin.py [hunk 4]
```diff
@@ -702,6 +704,25 @@ def command(cmd, **dargs):
     return ret
 
 
+def check_server_name(server_name="virtproxyd"):
+    """
+    Determine the server name under different daemon mode.
+    """
```
Comment (Yingshun) [inline virttest/virt_admin.py:710]: Same as above.

Comment (iccaszhulili) [reply virttest/virt_admin.py]: @Yingshun updated, thanks

File: virttest/virt_admin.py (hunk 3 above)
Comment (iccaszhulili) [reply virttest/virt_admin.py]: @Yingshun updated

**Review by Yingshun — APPROVED** [review]

File: virttest/virt_admin.py [hunk 5]
```diff
@@ -702,6 +704,32 @@ def command(cmd, **dargs):
...
+
+    :param server_name: name of the managed server
+    :return: name of the managed server
+    """
+    if not utils_split_daemons.is_modular_daemon():
+        server_name = "libvirtd"
+    return server_name
+
+
+def managed_daemon_config(conf_type="virtproxyd"):
```
Comment (chunfuwen) [inline virttest/virt_admin.py:719]: Could this be move into utils_config modular for more broader visibility ?

Comment (iccaszhulili) [reply virttest/virt_admin.py:719]: @chunfuwen The function just only useful for virt admin cases. 
# virt-admin server-list 
 Id   Name
------------------
 0    virtproxyd
 1    admin

As far as I know, the item of server name is not used in other features. It probably not very good to move this into utils_config

**Review by chunfuwen — APPROVED** [review]
LGTM

**Pull request merged**